src/views/Viewer.vue

```diff
@@ -852,9 +857,13 @@ export default {
 			}
 		}
 		::v-deep .modal-title,
-		::v-deep .modal-header .icons-menu button svg {
-			body.theme--light & {
-				color: var(--color-main-text) !important;
+		::v-deep .modal-header .icons-menu {
+			button, a {
+				svg {
+					body.theme--light & {
+						color: var(--color-main-text) !important;
```

Comment:
Hum, can we maybe make it prettier then and ensure we force that css on the entire header?

Reply:
I simplified it a bit, but don't really want to enforce that CSS for the entire header as it is modifying the color property